lib/marc/controlfield.rb

```diff
@@ -40,31 +40,26 @@ def initialize(tag,value='')
     # Two control fields are equal if their tags and values are equal.
 
     def ==(other)
-      if @tag != other.tag
-        return false 
-      elsif @value != other.value
-        return false
-      end
-      return true
+      tag == other.tag && value == other.value
     end
 
     # turning it into a marc-hash element
     def to_marchash
       return [@tag, @value]
```

Comment:
The `return ` is superfluous (reserve use of return when a method should cease execution early, prior to the last statement).  In this case, the last statement is the return value.  Ditto for other methods, like `to_hash`.